packages/pytest-simcore/src/pytest_simcore/services_api_mocks_for_aiohttp_clients.py

```diff
@@ -441,4 +447,28 @@ async def storage_v0_service_mock(
         repeat=True,
     )
 
+    aioresponses_mocker.post(
+        complete_upload_pattern,
+        status=web.HTTPOk.status_code,
+        payload={
+            "data": parse_obj_as(
+                FileUploadCompleteResponse,
+                {"links": {"state": _dummy_s3_url}},
+            ).dict()
+        },
+    )
+
+    aioresponses_mocker.post(
+        _dummy_s3_url,
+        status=web.HTTPOk.status_code,
+        payload={
+            "data": {"state": "ok", "e_tag": "my_etag"},
+        },
+    )
+
+    aioresponses_mocker.delete(
+        _dummy_s3_url,
+        status=web.HTTPOk.status_code,
+    )
```

Comment:
I think these are not the correct API that storage service provides:
please check here: https://github.com/ITISFoundation/osparc-simcore/blob/9cacf2db9d2ee4f9e471096695f7df96c3492c1a/services/storage/src/simcore_service_storage/handlers_files.py

in particular, abort_upload_file is defined as: 
```python
@routes.post(
    f"/{api_vtag}/locations/{{location_id}}/files/{{file_id}}:abort",
    name="abort_upload_file",
)
```

and complete_upload_file as
```python
@routes.post(
    f"/{api_vtag}/locations/{{location_id}}/files/{{file_id}}:complete",
    name="complete_upload_file",
)
```

Reply:
I have redone this part, so not I believe the paths should be correct, except perhaps for certain conventions about the etags. For those I just use UUIDs, but it could be they are actually something else.